lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -438,7 +443,11 @@ def build_entity(self):
             format=otypes.DiskFormat(
                 self._module.params.get('format')
             ) if self._module.params.get('format') else None,
-            sparse=self._module.params.get('format') != 'raw',
+            sparse=self._module.params.get(
+                'sparse'
+            ) if self._module.params.get(
+                'sparse'
+            ) else self._module.params.get('format') != 'raw',
```

Comment:
It should be `if self._module.params.get('sparse') is not None`, because if `sparse` is `False` it won't send `sparse=False`

Reply:
thanks!